login/Main.js

```diff
@@ -0,0 +1,12 @@
+const singIn=require("./SingIn.js");
+const LoginRequest  = require("./LoginRequest.js");
+const express = require("express");
+const app = express();
+
+
+let loginRequest = new  LoginRequest('Bob123','Bob123');
+
+app.listen(3000);
+app.get("/SingIn",function(request, response){
+    singIn.SingIn(loginRequest,request,response);
```

Comment:
Maybe it would be better to rename the signIn variable into signInService or something of the sort.

Reply:
I think about it.Fix is coming